src/main/org/codehaus/groovy/classgen/AsmClassGenerator.java

```diff
@@ -375,6 +375,13 @@ protected void visitConstructorOrMethod(MethodNode node, boolean isConstructor)
             visitParameterAnnotations(parameters[i], i, mv);
         }
 
+        // Add parameter names to the MethodVisitor (jdk8+ only)
+        if (getCompileUnit().getConfig().getParameters()){
```

Comment:
A very minor nit - there should be a space between `...getParameters())` and `{`